nrpy/c_codegen.py

```diff
@@ -650,103 +676,129 @@ def c_codegen(
 #    codegen to output our desired fabs().
 nrpyAbs = sp.Function("nrpyAbs")
 custom_functions_for_SymPy_ccode = {
```

Comment:
Let's move this dictionary definition into helpers/custom_c_codegen_functions.py .

Reply:
Resolved in e53593abe7ed8555382031859042a683b3cb35d9